scheduler/mesos.go

```diff
@@ -523,13 +528,22 @@ func (sched *VDCScheduler) FrameworkMessage(_ sched.SchedulerDriver, eid *mesos.
 func (sched *VDCScheduler) SlaveLost(_ sched.SchedulerDriver, sid *mesos.SlaveID) {
 	log.Errorln("slave lost: %v", sid)
 
-	agentMesosID := *sid.Value
-
 	ctx, err := model.Connect(context.Background(), sched.zkAddr)
 	if err != nil {
 		log.WithError(err).Error("Failed model.Connect")
 	}
 
+	agentMesosID := *sid.Value
+
+	crashed_node, err := model.CrashedNodes(ctx).FindByAgentMesosIDNotReconnected(agentMesosID)
```

Comment:
Couldn't the same result already be achieved by using the function ```FindByAgentMesosID``` and then just checking GetReconnected?
```
if crashed_node != nil  && crashed_node.GetReconnected() == false {
		// Node already added to crashed nodes. Nothing to do.
    return
}
```

Reply:
If a node crashes and reconnects multiple times, it will get a new entry in zookeeper for every crash. `FindByAgentMesosID` only returns the first entry found so that one might have Reconnected set to true while there's still another one with Reconnected set to false.

Reply:
Ah yeah, never mind then, I had forgotten that it only returns the first value ^^

Reply:
So, is it ok for merge then? :)

Reply:
Looks ok to me!